contracts/src/v0.8/automation/chains/OptimismModule.sol

```diff
@@ -16,7 +16,7 @@ contract OptimismModule is ChainModuleBase {
   uint256 private constant FIXED_GAS_OVERHEAD = 60_000;
   uint256 private constant PER_CALLDATA_BYTE_GAS_OVERHEAD = 270;
 
-  function getCurrentL1Fee() external view override returns (uint256) {
+  function getCurrentL1Fee(uint256) external view override returns (uint256) {
```

Comment:
using `uint256` will make it not accurate
using `bytes` will make it waste lots of gas

Reply:
1. passing bytes into this function
2. concat this bytes with a padding
3. passing the concated bytes to getL1Fee

with chain module (uint256) and V2: nothing
with chain module (bytes) and V2: 1 (BASE, OP), 1/2/3 (Scroll)